### PR TITLE
Some prepare work to support cvs job failover

### PR DESCRIPTION
**Pull request by hanfei1991**
This change is mainly in demo-server. Right now we read and write to kv-db in order to
1. read from middle
2. repeated write

### CODE REVIEW

File: executor/cvsTask/cvstask.go [hunk 1]
```diff
@@ -31,24 +30,25 @@ type strPair struct {
...
 type Config struct {
+	Idx      int    `json:"Idx"`
 	SrcHost  string `json:"SrcHost"`
-	SrcDir   string `json:"SrcDir"`
 	DstHost  string `json:"DstHost"`
-	DstDir   string `json:"DstDir"`
-	StartLoc int64  `json:"StartLoc"`
+	DstDir   string `json:"DstIdx"`
+	StartLoc string `json:"StartLoc"`
 }
```
Comment (amyangfei) [inline executor/cvsTask/cvstask.go:38]: Should we update the demo task config in `sample/config/demo.json`

Comment (hanfei1991) [reply executor/cvsTask/cvstask.go:38]: Good catch. Let me think about how to launch demo job by master-client ...

Comment (hanfei1991) [reply executor/cvsTask/cvstask.go:38]: Finished, We can still use master-client to test locally, which need to add flags to demo-server, such as `./bin/demoserver -d /datax -a 0.0.0.0:1234 -r 1000000`  PTAL

File: cmd/demoserver/kvdata_test.go [hunk 2]
```diff
@@ -11,21 +13,30 @@ import (
...
 
-const (
-	demoAddr = "127.0.0.1:1234"
-	dirName  = "/tmp/df_test"
-)
-
 func TestDemoLogic(t *testing.T) {
+	DemoAddress = "127.0.0.1:1234"
+	DemoDir = "/tmp/data"
+	WtDir := "/tmp/data1"
```
[the remark this answers is not in the document]
Comment (hanfei1991) [reply cmd/demoserver/kvdata_test.go:19]: This dirs are created by pkg/db/peddle.go in every "openPeddle", no matter it is created or not. 
Is it still nesessary?

**Review by amyangfei — APPROVED** [review]

**Pull request merged**